docs/index.rst

```diff
@@ -23,6 +23,24 @@ sunkit-image
           generated/gallery/index
           changelog
 
+Mission Statement
+=================
+
+The goal of the ``sunkit-image`` library is to provide access to image processing routines that are:
+
+1. Focused on being applied to solar physics data.
+2. Are published in the literature or in preparation to be published.
+3. If there is no publication, it must be widely used throughout the solar physics community.
+   One example might be co-alignment routines that exist in SSWIDL.
```

Comment:
```suggestion
   One example might be co-alignment routines that exist in SSWIDL.
```
All possible coalignment routines in SSW? Do we need this example?

Reply:
I didn't have a good example, so I came up with this.